xml/transactional_updates.xml

```diff
@@ -0,0 +1,444 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" version="5.0"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xml:id="cha.transactional-updates" xml:lang="en">
+ <title>Transactional Updates</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Transactional updates are available in &productname; as a technology
+    preview, for updating &slsa; when the root filesystem is read-only. 
```

Comment:
In general: transactional update should be used, if disturbing a running service is more expensive than a reboot. If a reboot is very expensive, transactional updates should not be used.